features/attached_enable.feature

```diff
@@ -225,20 +226,22 @@ Feature: Enable command behaviour when attached to an UA subscription
 
     @series.focal
     @uses.config.machine_type.lxd.vm
-    Scenario: Attached enable of vm-based services in a focal lxd vm
+    Scenario Outline: Attached enable of vm-based services in a focal lxd vm
```

Comment:
@lucasmoura what is your opinion about avoiding a Scenario Outline for this test and just leaving the replicated tests? The reason I ask, is because this a non-destructive test to the VM (nothing changes config-wise when we pass) and it costs us the setup of a brand new VM to just test the other case